src/test/scala/com/komanov/uuid/GuidParserPerformanceTest.scala

```diff
@@ -47,7 +48,8 @@ object GuidParserPerformanceTest extends App {
   doTest("java 3  ", javaFast3F)
   doTest("java 4  ", javaFast4F)
   doTest("java 5  ", javaFast5F)
-  doTest("java f  ", javaFast5F)
+  doTest("java f  ", javaFastFF)
```

Comment:
you forgot to test your final method, you just retested the 5th version instead.